### PR TITLE
fix: translations

### LINKED ISSUE #263: Find strings in haiku
## User Story
As product manager I want to be able to edit all the texts on GameDAO.co so I can either change them or translate them.

## Acceptance Criteria

1. view all files for strings which are not yet represented by a translation key 
2. also include features configurations from GraphQL 
3. create a translation key for those strings.

## Additional Information

- inform product manager when all strings can be translated via weblate

## How to test

1. all texts can be edited in Weblate

### PR DESCRIPTION
**Pull request by developer-sidani**
### closes #263

### CODE REVIEW

File: pkg/app/src/components/Account/modules/IdentitySection/form.tsx [hunk 1]
```diff
@@ -218,7 +214,7 @@ export function IdentityForm() {
 											}}
 											disabled
 											value="1.0000"
-											label="Total Deposit"
+											label={t('label:total_deposit')}
 											InputProps={{
 												endAdornment: <Typography>milli</Typography>,
```
Comment (PAkhidue) [inline pkg/app/src/components/Account/modules/IdentitySection/form.tsx:219]: Should be translated as well

File: pkg/app/src/components/Account/modules/IdentitySection/form.tsx [hunk 2]
```diff
@@ -100,7 +96,7 @@ export function IdentityForm() {
 									render={({ field: { onChange, value }, formState: { errors } }) => (
 										<TextField
 											fullWidth
-											label="Display Name"
+											label={t('label:display_name')}
 											placeholder="QDozer"
```
Comment (PAkhidue) [inline pkg/app/src/components/Account/modules/IdentitySection/form.tsx:100]: Translation here as well

**Review by PAkhidue — CHANGES_REQUESTED** [review]
Also in the balanceCard.tsx there is a missing "Balances" translation.
There are also some <Layout> components called with title as a string instead of translation. (noWalletConnected.tsx & organisations/index.tsx)

Didn't check everything yet but the most things are translated. Nice :)

File: pkg/app/src/components/Account/modules/IdentitySection/form.tsx (hunk 2 above)
Comment (developer-sidani) [reply pkg/app/src/components/Account/modules/IdentitySection/form.tsx:100]: @PAkhidue placeHolder is an example of a display name, translation is done here!

**Review by PAkhidue — APPROVED** [review]

**Pull request merged**